src/config.rs

```diff
@@ -398,4 +400,46 @@ mod tests {
             }
         );
     }
+
+    #[test]
+    fn mentions_config() {
+        let config = r#"
+        [mentions."some_other_path"]
+        message = "foo"
+        cc = ["@someone"]
+
+        [mentions]
+        bors-commit-message = "has bors commit"
+        "#;
+        let config = toml::from_str::<Config>(config).unwrap().mentions.unwrap();
+        assert!(config.paths.len() == 1);
+        assert_eq!(
+            config.bors_commit_message.as_deref(),
+            Some("has bors commit")
+        );
+
+        let config = r#"
+        [mentions]
+        bors-commit-message = "has bors commit"
+
+        [mentions."some_other_path"]
+        message = "foo"
+        cc = ["@someone"]
+        "#;
+        let config = toml::from_str::<Config>(config).unwrap().mentions.unwrap();
+        assert!(config.paths.len() == 1);
+        assert_eq!(
+            config.bors_commit_message.as_deref(),
+            Some("has bors commit")
```

Comment:
How could this middle test possibly fail?

This test should be over the overall output, not the config system